tests/utils/gist-spec.ts

```diff
@@ -31,5 +31,10 @@ describe('gist', () => {
       const result = urlFromId(mockId);
       expect(result).toBe(`https://gist.github.com/${mockId}`);
     });
+
+    it('returns an empty string if id is undefined', () => {
+      const result = urlFromId(undefined);
```

Comment:
It's still right but can be also used without providing `undefined`, leave that for you

```suggestion
      const result = urlFromId();
```